static/js/components/User.js

```diff
@@ -34,43 +29,13 @@ export default class User extends React.Component {
   render() {
     const { profile } = this.props;
 
-    let getStateName = () => {
-      if ( profile.state_or_territory ) {
-        return iso3166.subdivision(profile.state_or_territory).name;
-      } else {
-        return '';
-      }
-    };
-
-    return <div className="card">
+    return <div>
```

Comment:
please put `className="single-column"` on either this div or a wrapping `div` in `UserPage`. You should be able to delete most of the classes you have (like `profile-form-center-card`) when you do that.